Common/models/playerteam.model.ts

```diff
@@ -1,15 +0,0 @@
-import {BaseEntity, Column, Entity, ManyToOne, OneToMany, PrimaryColumn} from 'typeorm';
-import PlayerModel from './player.model';
-
-@Entity({ name: "player_team_model" })
-export default class PlayerTeamModel extends BaseEntity {
-
-    @ManyToOne(() => PlayerModel)
-    player: PlayerModel;
-    @PrimaryColumn('text')
```

Comment:
These should also be foreign keys like how player is. Player should also have 2 columns, one for the model and one for the ID.